examples/meshdirectory/static/index.html

```diff
@@ -76,12 +76,13 @@ <h5>Service description:</h5>
         },
         methods: {
             providerLink(provider) {
-                const query = this.queryParams.toString() ? `?${this.queryParams}` : ''
-                return `${this.ocmService(provider).endpoint.path}${query}`
+                const query = this.queryParams.toString() ? `?${this.queryParams}` : '';
+                const inviteApi = `${this.ocmService(provider).endpoint.path}/invites/forward`;
```

Comment:
Can we use ``const inviteApi = new URL('/invites/forward', `${this.ocmService(provider).endpoint.path}`);``? That'll take care of double slashes.

Reply:
@ishank011, I tried that initially, but that seems to always take away any path that may be already present in the provider endpoint url (`/iop/ocm`) and replace it with `/invites/forward`. I'll try to take care of that :)

Reply:
Oh okay. We can skip it then. No need to put too much effort into this